test/unit/io/alignment_file/format_sam_test.cpp

```diff
@@ -136,24 +585,24 @@ TEST_F(alignment_data, default_options_all_members_specified)
 
     std::string comp
     {
-        "read1\t41\tref\t1\t61\t1S1M1D2M\tref\t10\t300\tACGT\t!##$\tAS:i:2\tNM:i:7\n"
+        "read1\t41\tref\t1\t61\t1S1M1D1M1I\tref\t10\t300\tACGT\t!##$\tAS:i:2\tNM:i:7\n"
         "read2\t42\tref\t2\t62\t7M1D1M1S\tref\t10\t300\tAGGCTGNAG\t!##$&'()*\txy:B:S,3,4,5\n"
-        "read3\t43\tref\t3\t63\t1S1M1D4M1D1M1S\tref\t10\t300\tGGAGTATA\t!!*+,-./\n"
+        "read3\t43\tref\t3\t63\t1S1M1D1M1I1M1I1D1M1S\tref\t10\t300\tGGAGTATA\t!!*+,-./\n"
     };
 
     for (unsigned i = 0; i < 3; ++i)
-        /*EXPECT_NO_THROW(( */format.write(ostream, options, header_ptr,
-                                       seqs[i], quals[i], ids[i], offsets[i], std::string{},
-                                       ref_id, ref_offsets[i], alignments[i],
-                                       flags[i], mapqs[i], mates[i],
-                                       tag_dicts[i], 0u, 0u)/*))*/;
+        ASSERT_NO_THROW(format.write(ostream, options, header_ptr,
```

Comment:
why assert and not expect?

Reply:
Convenience... In some tests I get a mirade of errors in the EXPECT_EQ tests afterwards because on error the variables have not been filled. Smaller tests also have it for consistency.

Reply:
Hm, we are using EXPECT everywhere else until now. It's not super important, but maybe should discuss this at a strategy meeting?

Reply:
Yes we have EXPECT anywhere else so far but I think it is very handy to only show "the first and most important error" when I know that all of the expect's afterwards will be false anyway. I will add a card in for the next strategy meeting but would really like to keep this here as long as we have not decided against it.

Reply:
Ok